### PR TITLE
fix call to for_read in auth docs

### PR DESCRIPTION
**Pull request by barnabasJ**

### CODE REVIEW

File: documentation/topics/security/actors-and-authorization.md [hunk 1]
```diff
@@ -29,13 +29,15 @@ MyDomain.create_post!(Post, authorize?: true)
 ```
 
 > ### Set the actor on the query/changeset/input {: .warning}
+>
 > The hooks on a query/changeset/input to an action may need to know the actor, so you
 > need to set the actor when building them, not when calling the action.
+>
```
Comment (barnabasJ) [inline documentation/topics/security/actors-and-authorization.md:35]:
```suggestion
```

File: documentation/topics/security/actors-and-authorization.md [hunk 2]
```diff
@@ -29,13 +29,14 @@ MyDomain.create_post!(Post, authorize?: true)
 ```
 
 > ### Set the actor on the query/changeset/input {: .warning}
+>
```
Comment (barnabasJ) [inline documentation/topics/security/actors-and-authorization.md:32]:
```suggestion
```

**Review by zachdaniel — APPROVED** [review]

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**